index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="Wilhelm" content="Kamulunga">
+  <title>My Youtube Page</title>
+  <link href="styles.css" rel="stylesheet">
+</head>
+<body>
+  <header>
+    <h1>My Youtube Page</h1>
+  </header>
+  <div class="form">
+    <form id="search-form" class="style-scope ytd-searchbox" action="/results">
+      <div id="container" class="style-scope ytd-searchbox">
+        <yt-icon id="search-icon" class="style-scope" ytd-searchbox>
+        </yt-icon>    
+      </div>
+    </form>
+  </div>
+  <div class="container">
```

Comment:
- Because of semantic reasons, I suggest wrapping this section with a <section> tag is better than with a <div> tag. Like this

<section class="container">

       ......

</section>